dll/win32/shell32/lang/cs-CZ.rc

```diff
@@ -11,11 +11,11 @@ LANGUAGE LANG_CZECH, SUBLANG_DEFAULT
 MENU_001 MENUEX
 BEGIN
     MENUITEM "&Vedle sebe", FCIDM_SHVIEW_BIGICON
-    MENUITEM "&Ikony", FCIDM_SHVIEW_SMALLICON
+    MENUITEM "&Ikony", FCIDM_SHVIEW_SMALLICON //FIXME: accelerator collision &I
     MENUITEM "&Seznam", FCIDM_SHVIEW_LISTVIEW
     MENUITEM "&Podrobnosti", FCIDM_SHVIEW_REPORTVIEW
     MENUITEM "", -1, MFT_SEPARATOR
-    POPUP "Seřadit &ikony", FCIDM_SHVIEW_ARRANGE
+    POPUP "Seřadit &ikony", FCIDM_SHVIEW_ARRANGE //FIXME: accelerator collision &I
```

Comment:
A suggestion would be to put the accelerator on the "e" of Seradit.

Reply:
I know, but I will leave that to the native speakers, unless you would know exactly where it has to be placed for czech.